src/core/CoolBoardSensors.cpp

```diff
@@ -192,8 +192,8 @@ float CoolBoardSensors::readVBat() {
   // read battery voltage
   int raw = analogRead(A0);
   // convert it to approx. correct tension in volts
-  float val = 6.04 / 1024 * raw;
-
+  float val = 5.47 / 1023.0 * raw;
```

Comment:
Why ? Those are magic numbers. Why 6.04 ? Why 1023 ? Why both ?

Reply:
ah and divide by 1024!